planner/core/logical_plan_builder.go

```diff
@@ -1111,7 +1111,7 @@ func (a *havingWindowAndOrderbyExprResolver) Leave(n ast.Node) (node ast.Node, o
 				}
 			}
 		}
-		index := -1
+		var index int
```

Comment:
I think we should keep the original initialization value.

Reply:
These original initialization values will be overwritten in the following code.